modules/admin_manual/pages/configuration/server/occ_commands/app_commands/_ldap_integration_commands.adoc

```diff
@@ -5,24 +5,48 @@ Marketplace URL: {oc-marketplace-url}/apps/user_ldap[LDAP Integration]
 [source,console]
 ----
 ldap
- ldap:check-user               Checks whether a user exists on LDAP.
+ ldap:check-user               Checks whether a user exists on LDAP
  ldap:create-empty-config      Creates an empty LDAP configuration
  ldap:delete-config            Deletes an existing LDAP configuration
+ ldap:invalidate-cache         Invalidates the LDAP cache
  ldap:search                   Executes a user or group search
  ldap:set-config               Modifies an LDAP configuration
  ldap:show-config              Shows the LDAP configuration
  ldap:test-config              Tests an LDAP configuration
 ----
 
+== Search for a User
+
 Search for an LDAP user, using this syntax:
 
 [source,console,subs="attributes+"]
 ----
-{occ-command-example-prefix} ldap:search [--group] [--offset="..."] [--limit="..."] search
+{occ-command-example-prefix} ldap:search [options] [--] <search>
 ----
+=== Arguments
+
+[width="100%",cols="20%,70%",]
+|===
+| `search`
+| The search string operates against the users fullname (`cn`). It is a by default a prefix search.
```

Comment:
> The search string operates against the users fullname (`cn`)

I'm quite sure this is by default. It's configurable in the ldap wizard; there is a section to use search attributes in the advanced tab. In fact, it uses the configured displayname attribute, which happens to be the "cn".

It could be something like:
Search attributes -> ownCloud's displayname attribute (by default) -> "cn" (by default)